torch/ao/quantization/pt2e/eval_utils.py

```diff
@@ -2,19 +2,21 @@
 import torch.nn.functional as F
 
 
-def _replace_dropout_for_eval(m: torch.fx.GraphModule):
+def _replace_dropout(m: torch.fx.GraphModule, for_eval_mode: bool):
```

Comment:
nit: `for_eval_mode` might be a bit vague I feel, maybe just "train_to_eval" and explain in the comment saying when this is False it means eval_to_train?